tests/Unit/Provider/Command/MessageSendTest.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCA\Mail\Tests\Unit\Provider\Command;
+
+use ChristophWurst\Nextcloud\Testing\TestCase;
+use OCA\Mail\Account;
+use OCA\Mail\Db\LocalAttachment;
+use OCA\Mail\Db\LocalMessage;
+use OCA\Mail\Db\MailAccount;
+use OCA\Mail\Provider\Command\MessageSend;
+use OCA\Mail\Service\AccountService;
+use OCA\Mail\Service\Attachment\AttachmentService;
+use OCA\Mail\Service\OutboxService;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\IConfig;
+use OCP\Mail\Provider\Address;
+use OCP\Mail\Provider\Attachment;
+use OCP\Mail\Provider\Exception\SendException;
+use OCP\Mail\Provider\Message;
+
+class MessageSendTest extends TestCase {
+
+	/** @var IConfig&MockObject */
+	private $config;
+	/** @var ITimeFactory&MockObject */
+	private $time;
+	/** @var AccountService&MockObject */
+	private $accountService;
+	/** @var OutboxService&MockObject */
+	private $outboxService;
+	/** @var AttachmentService&MockObject */
+	private $attachmentService;
+	/** @var AccountService&MockObject */
```

Comment:
```suggestion
	/** @var AccountService */
```

the SUT is not a mock

Reply:
You phpdoc-typed $commandSend as AccountService

Reply:
Now I understand.

Reply:
Done